### PR TITLE
fix: lifting action section

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-913](https://emurgo.atlassian.net/browse/YOMO-913)

[YOMO-913]: https://emurgo.atlassian.net/browse/YOMO-913?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jorbuedo — APPROVED** [review]

**Review by SorinC6 — APPROVED** [review]

File: apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx [hunk 1]
```diff
@@ -54,73 +54,69 @@ export const DelegationConfirmation = () => {
 
   return (
     <View style={styles.container}>
```
Comment (stackchain) [inline apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx:56]: is it a screen? if it is it should be wrapped within SafeAreaView.

File: apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx [hunk 2]
```diff
@@ -54,73 +54,69 @@ export const DelegationConfirmation = () => {
...
+          onSuccess={onSuccess}
+          setUseUSB={setUseUSB}
+          useUSB={useUSB}
+          yoroiUnsignedTx={yoroiUnsignedTx}
+          chooseTransportOnConfirmation
+        />
+      </Actions>
+
+      {/* hack to fix weird KeyboardAvoidingView bug in THIS SCREEN */}
+      {Platform.OS === 'ios' && <KeyboardSpacer />}
```
Comment (stackchain) [inline apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx:119]: maybe we should start using View.onLayout  (thinking out loud, ignore it for now)

File: apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx (hunk 1 above)
Comment (stackchain) [reply apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx:56]: Maybe this is the issue?

Comment (banklesss) [reply apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx:56]: just checking and unfortunately the issue still persists

Comment (banklesss) [reply apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx:56]: also I tested using paddings instead of margins

Comment (banklesss) [reply apps/wallet-mobile/src/Staking/DelegationConfirmation/DelegationConfirmation.tsx:56]: The problem is that the button stays behind the keyboard until you click to edit the input. When you start editing, the button goes into place.

I tried to force the focus but it didn't work either.

**Pull request merged**